apps/web/src/scenes/NftDetailPage/NftDetailAsset.tsx

```diff
@@ -185,10 +189,18 @@ function NftDetailAssetComponentWithouFallback({
 type Props = {
   tokenRef: NftDetailAssetFragment$key;
   hasExtraPaddingForNote: boolean;
-  visibility?: string; // prop to pass the visibility state of the selected NFT
+  visibility?: string; // whether or not the nft is currently the visible "slide" in the detail page carousel
+  toggleLightbox: () => void;
+  showLightbox?: boolean;
```

Comment:
nit: `showLightbox` sounds like a callback function like `toggleLightbox`, as opposed to the boolean that it is. could we make this `shouldShowLightbox` or `isLightboxShown`?